test/Main.purs

```diff
@@ -0,0 +1,128 @@
+module Test.Main where
```

Comment:
@cryogenian thank you! :smile: Might be useful to do something like this for most of the standard type classes and factor it out so that it can be deployed in different libraries...